libs/eo/transfers/src/lib/test-data.ts

```diff
@@ -17,85 +17,85 @@
 export const testData = [
   {
     id: '99f96d44-494b-439f-83fd-8846b13d22ad',
-    startDate: new Date().setDate(new Date().getDate() - 60),
-    endDate: new Date().setDate(new Date().getDate() + 60),
+    startDate: '2023-06-08T22:00:00+00:00',
```

Comment:
The API should return all timestamps in Unix Time Seconds. Is this a mock of what is returned from the API? If it is, then it should have timestamps in Unix Time Seconds

Reply:
Removed this file, and confirmed working with timestamps being returned instead of strings